_layouts/doc.html

```diff
@@ -7,10 +7,21 @@
     <div class="row">
       <ol class="breadcrumb">
         <li><a href="{{site.baseurl}}/">Home</a></li>
-        {% for bc in page.breadcrumbs %}
-        <li><a href="{{site.baseurl}}/{{bc}}">{{bc}}</a></li>
+        {% assign splitted_url = page.url | split:"/" %}
+        {% assign repo_name = splitted_url[2] %}
+        {% for repo in site.docs_repos %}
```

Comment:
@apojomovsky in light of this, should we consider making `docs_repos` a hash instead of an array?

Reply:
great idea! just did as suggested